tests/dummy/app/locales/en/translations.js

```diff
@@ -251,7 +251,9 @@ Ember.$.extend(true, translations, {
       'birthday-caption': 'Birthday',
       'gender-caption': 'Gender',
       'vip-caption': 'VIP',
-      'karma-caption': 'Karma'
+      'karma-caption': 'Karma',
+      'vname-caption': 'Name is required',
```

Comment:
v<field-name>-caption is not better key name for validation translation.
Rename it please to something like name-validation-message-caption.
And other similar translations keys too (dont't forget to rename them in model's validation options).